src/openvpn/multi.h

```diff
@@ -613,6 +613,10 @@ multi_get_timeout(struct multi_context *m, struct timeval *dest)
 static inline bool
 multi_process_outgoing_tun(struct multi_context *m, const unsigned int mpp_flags)
 {
+#ifdef ENABLE_VLAN_TAGGING
+    void multi_prepend_8021q_vlan_tag (const struct context *c,
+                                             struct buffer *buf);
+#endif
```

Comment:
Declaring a function inside another is a style thing that we are trying to get rid of. Please, declare it outside like the other prototypes.